build.gradle.kts

```diff
@@ -138,11 +138,19 @@ subprojects {
             }
         }
     }
-
-    plugins.withType<LibraryPlugin> {
-        tasks.register<Jar>(sourcesTaskName).configure {
-            archiveClassifier.set("sources")
-            from(this@withType.extension.sourceSets["main"].java.srcDirs)
+    plugins.withId("com.android.library") {
```

Comment:
Is type -> string weakening necessary here?

Reply:
No, we can replace by Type